tests/test_tools.py

```diff
@@ -35,8 +35,8 @@ def test_coordinate_identity_transform_rectangular(self):
 
         z = po.tools.polar_to_rectangular(*po.tools.rectangular_to_polar(torch.complex(x, y)))
 
-        assert torch.norm(x - z.real) < 1e-3
-        assert torch.norm(y - z.imag) < 1e-3
+        assert torch.linalg.norm(x - z.real) < 1e-3
+        assert torch.linalg.norm(y - z.imag) < 1e-3
```

Comment:
```suggestion
        assert torch.linalg.vector_norm((x - z.real).flatten(), ord=2) < 1e-3
        assert torch.linalg.vector_norm((y - z.imag).flatten(), ord=2) < 1e-3
```
Let's make this a bit more readable, since this is behavior that torch is doing.